cmd/state-installer/internal/installer/installation_test.go

```diff
@@ -24,6 +26,7 @@ var updatedTestFileContent []byte = []byte("#!/bin/bash\necho updated;")
 func init() {
 	if runtime.GOOS == "windows" {
 		stateToolTestFile = "state.exe"
+		stateTrayTestFile = "state-tray.exe"
 		otherTestFile = "other.exe"
 		installerTestFile = "state-installer.exe"
```

Comment:
You could use this in the definitions above, then drop this whole block:

http://github.com/ActiveState/cli/blob/dd597af58d16cdac0f3cf0694338598543ec15eb/internal/exeutils/exeutils_windows.go#L3-L3